.gitignore

```diff
@@ -0,0 +1,3 @@
+pqcli.egg-info
+build
+$HOME/
```

Comment:
This really shouldn't be necessary. Is this an artifact from Windows?

Reply:
you mean the $HOME ? windows has %% variables, so $HOME is created automatically when you run pqcli from git folder

Reply:
That's another issue, it should be using something like `%APPDATA%` on Win platform